### ISSUE TITLE
cartopy not working on Windows, neither with python 3.10 nor with python 3.11

### ISSUE THREAD

**Issue by thomas-haslwanter** [SciTools/cartopy#2205]
### Description
Cannot get cartopy to work on Windows, neither with python 3.10, nor with 3.11

When trying to install the current version, I get the error

`lib/cartopy/trace.cpp(767): fatal error C1083: Cannot open include file: 'geos_c.h': No such file or directory
      error: command 'C:\\Program Files (x86)\\Microsoft Visual Studio\\2019\\Community\\VC\\Tools\\MSVC\\14.29.30133\\bin\\HostX86\\x64\\cl.exe' failed with exit code 2`

And when I use the installable version of cartopy (0.20.2), then I get on `import cartopy` the error message

`    import cartopy.trace
ImportError: DLL load failed while importing trace: The specified module could not be found.
`

### Operating system
Win 3.11

### Cartopy version
0.20, and 0.21

### pip list

```
➜ pip show cartopy
Name: Cartopy
Version: 0.20.2
Summary: A cartographic python library with Matplotlib support for visualisation
Home-page: https://scitools.org.uk/cartopy/docs/latest/
Author: UK Met Office
Author-email:
License: LGPLv3
Location: c:\programs\wpy64-310110\python-3.10.11.amd64\lib\site-packages
Requires: matplotlib, numpy, pyproj, pyshp, shapely
Required-by: geoviews

**Comment by rcomer** [comment]
I believe #2197 will be the fix for all of these sorts of problems.

**Comment by greglucas** [comment]
We have just released v0.22 which should help with the compatibility between packages and installation much easier. Please open a new issue if you are still having problems.
